robottelo/hosts.py

```diff
@@ -389,7 +389,7 @@ def install_katello_ca(self, satellite):
             f'curl --insecure --output katello-ca-consumer-latest.noarch.rpm \
                     {satellite.url_katello_ca_rpm}'
         )
-        self.execute('rpm -Uvh katello-ca-consumer-latest.noarch.rpm')
+        self.execute('rpm -Uvh --nodigest --nofiledigest katello-ca-consumer-latest.noarch.rpm')
```

Comment:
Does the host have a property you can check, as it appears this workaround should only be applied when using FIPs.

@JacobCallahan thoughts on the impact if we stop having rpm validate the digest of the katello RPM? To me it seems this is going to far applied this broadly.

Reply:
fair point, I'll look into it

Reply:
I believe this has been addressed.